python/ray/data/_internal/stats.py

```diff
@@ -1099,6 +1122,20 @@ def __str__(self) -> str:
                 node_count_stats["mean"],
                 node_count_stats["count"],
             )
+        if output_num_rows_stats and self.time_total_s and wall_time_stats:
+            total_num_out_rows = output_num_rows_stats["sum"]
+            out += indent
+            out += "* Operator throughput:\n"
+            out += (
+                indent + "    * Ray Data throughput:"
```

Comment:
nit: could consider using `\t` for the tabbing instead of spaces to keep consistent with `indent` definition